transformations/summarization_transformation/transformation.py

```diff
@@ -0,0 +1,212 @@
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import spacy
+
+"""
+The code is partially adapted and modified from https://github.com/NSchrading/intro-spacy-nlp
+"""
+
+
+class Summarization(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+    ]
+    languages = ["en"]
+
+    def __init__(self):
+        # super().__init__(seed, max_outputs=max_outputs)
+        self.nlp = spacy.load('en_core_web_sm', disable=['ner','textcat'])
```

Comment:
You might want to use spacy like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/d39f0e5f57a79937296351996617ac03ee80fb74/transformations/close_homophones_swap/transformation.py#L55).

Reply:
Thanks for the suggestions. I have modified it.